specs/dotsigstore_bundle/pb/bundle.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package sigstore;
+
```

Comment:
Can we version the proto?

Reply:
I'm not familiar with any version support by protobuf itself. 
The message does contain a `media_type` which should be relied upon (especially during JSON serialization) for versioning, but this would be a manual step. Beyond that it's a bit harder. Protobuf is extensible without causing breaking changes to existing/older clients. What could be done is to version the generated source code packages created for each language, and so create a crude versioning schema. But I have no experience doing so, and not sure if it's gonna work for all languages.

Reply:
IIUC Hayden was referring to a manual step. So we could do something sem-ver-ish (add new features but no breaking changes in a minor version) and client libraries would be able to respect that.

Reply:
I added a comment on versioning, is that sufficient and we can resolve this?